### DISCUSSION TITLE
https://api.etsy.com/v3/application/shops/{shopId}/receipts returns HTTP 500 or invalid/empty JSON

### DISCUSSION THREAD

**Discussion by byter325** [etsy/open-api#1181] [Q&A]
A request to the receipts endpoint returns empty or invalid JSON or internal server error 500

**Comment by pawcik** [comment]
In my case, proper request to listings api once returns valid response and next time I got a 200 with empty content. 

My first guess was that maybe I hit the rate limit (which will be odd, since I'm not doing more than 20 request per day). But for the responses with content, the rate limiting headers looks ok:

```
X-Limit-Per-Day: 20000
X-Limit-Per-Second: 20
X-Remaining-This-Second: 19
X-Remaining-Today: 19977
```

My temporary workaround is the retry request once I got a response with content.
